src/benchmarklib/benchmark_table_encoder.cpp

```diff
@@ -136,6 +136,10 @@ bool BenchmarkTableEncoder::encode(const std::string& table_name, const std::sha
   }
   Hyrise::get().scheduler()->schedule_and_wait_for_tasks(jobs);
 
+  // Note: Pruning statistics might have already been generated during chunk encoding, in which case this call
+  // is (almost) a NoOp.
+  // However, this does not apply when loading benchmark data that we have cached, because that data is encoded already.
+  // We still need to generate the statistics in that case, though, because we don't cache them.
```

Comment:
Can you use "store binary tables" instead of caching? It is is some kind of caching, but written to disk. I think being more explicit helps here. Also, binary tables MIGHT be encoded correctly already, they might also be encoded with the "wrong" encoding.

And ... "do not" instead of "don't". :)